cmd/makeitrain.go

```diff
@@ -10,14 +10,17 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var dryRun bool
+var (
```

Comment:
Formatting looks weird

Reply:
I was trying to adjust help messages alignment on screen. I'm eager to hear your thoughts on this too.

Reply:
I think you replied to the wrong comment here 😅  I was referring to the alignment of the variable declarations. However I just checked and they look fine on my machine, but in git the alignment is inconsistent, so all good 😄